src/util.js

```diff
@@ -0,0 +1,24 @@
+const BASE_URL = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+export function parseId(result = '') {
+  const startIndex = 'Game with ID: '.length - 1;
+  const endIndex = result.lastIndexOf(' ');
+  return result.substring(startIndex, endIndex + 1);
+}
+
+export async function post(URL, body) {
+  const response = await fetch(`${BASE_URL}${URL}`, {
+    method: 'POST',
+    body: JSON.stringify(body),
+    headers: {
+      'Content-type': 'application/json; charset=UTF-8',
+    },
+  });
+
+  return response.json();
+}
+
+export async function get(URL) {
+  const response = await fetch(`${BASE_URL}${URL}`);
+  return response.json();
+}
```

Comment:
- [ ] Kindly change all these to `es6 arrow functions`. It keeps the project in line with recent trends.

Reply:
@mikenath223 I don't understand how this is blocking my PR from being approved. It's not part of the general requirement, and also not part of the Microverse best practice.